Gemfile.lock

```diff
@@ -1315,8 +1315,10 @@ GEM
     method_source (0.9.2)
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2021.0225)
-    mimemagic (0.3.6)
+    mime-types-data (3.2021.0212)
+    mimemagic (0.3.9)
```

Comment:
For reference please check below link
https://github.com/mimemagicrb/mimemagic/issues/98